app_spec/test/spawn_conductors.js

```diff
@@ -0,0 +1,81 @@
+const child_process = require('child_process')
```

Comment:
This does feel a bit awkward, now that we're at this point, of duplicating most of this code... roughly in three places now `hc_cluster_test`, here, and in `diorama`

I wonder if we can do some easy deduping

Reply:
Diorama: yes, but we are not using Diorama here anymore. Try-o-rama does not spawn conductors anymore. (Though this might change with the next step).

Concerning cluster-test, let's talk about that. Would cluster-test at some piont be completely merged with app-spec? If not, yes we should at least merge the spawn_conductor code. Copied it here because I was to lazy to setup TypeScript for app-spec..